priv/repo/seeds.exs

```diff
@@ -24,3 +25,131 @@ Enum.each(0..100, fn x ->
   |> Meeting.changeset(%{date: meeting_date})
   |> Repo.insert!()
 end)
+
```

Comment:
We do not want to have speakers seeded in the app. The reason being in that we have the see file setting up the meeting dates for us. We would run this in the event we have an issue with our server and needed to setup the dates again. Seeding speakers would put a lot of bogus data in the DB at the time we setup our dates again if we needed to do that.

Reply:
If you are needing Speakers to test this with you can either A) create new speakers using the UI at `speakers/new` (you would just need to comment out the basic auth in the speaker_controller to get past the authentication, or B) Add some speakers to the DB via the command line.